lv_conf_template.h

```diff
@@ -119,6 +119,16 @@
     * 0: to disable caching */
     #define LV_CIRCLE_CACHE_SIZE 4
 
+    /*Allow dithering gradient (to achieve visual smooth color gradients on limited color depth display)
+    *LV_DITHER_GRADIENT implies consuming one or two lines of the object's rendering surface
```

Comment:
How do you mean `consuming`?

Reply:
sorry, french word. I meant "using as much memory as"